scripts/normalize-by-median.py

```diff
@@ -55,80 +48,94 @@ def WithDiagnostics(ifile, batch_size, fp, paired, norm):
 
     index = 0
 
-    for index, batch in enumerate(batchwise(
-                                  screed.open(ifile, parse_description=False),
-                                  batch_size)):
+    # per read diagnostic output
+    for index, is_paired, read0, read1 in reader:
 
-        norm.total += batch_size
-        total = norm.total
-        discarded = norm.discarded
+        if is_paired:
```

Comment:
Please move this accounting into the Normalizer object.  No need to mess with norm.total here.